File: 백준/Bronze/11295. Exercising/Exercising.c
#include <stdio.h>
int main(void) {
    int L, n, walk, cnt = 1;
    while (1) {
        scanf("%d", &L);
        if (L == 0) break;
        scanf("%d", &n);
        printf("User %d\n", cnt++);
        for (int i = 0; i < n; i++) {
            scanf("%d", &walk);
            printf("%.5f\n", walk * L / 100000.0);
        }
    }
    return 0;
}

File: 백준/Bronze/11295. Exercising/Exercising.py
import sys
input=lambda:sys.stdin.readline().rstrip()
cnt=1
while True:
    L=int(input())
    if L==0: break
    n=int(input())
    print("User %d" % cnt)
    for _ in range(n):
        walk=int(input())*L
        print("%.5f" % (walk/100000))
    cnt+=1